tests/test_search.py

```diff
@@ -312,20 +308,16 @@ def test_median_stamps_trj(self):
                 pix_values0.append(pixVal)
             if pixVal != KB_NO_DATA and goodIdx[1][i] == 1:
                 pix_values1.append(pixVal)
-        self.assertEqual(len(pix_values0), self.imCount)
-        self.assertEqual(len(pix_values1), self.imCount - 3)
+        self.assertEqual(len(pix_values0), self.img_count)
+        self.assertEqual(len(pix_values1), self.img_count - 3)
 
         # Check that we get the correct answer.
         self.assertAlmostEqual(np.median(pix_values0), medianStamps0.get_pixel(2, 2), delta=1e-5)
         self.assertAlmostEqual(np.median(pix_values1), medianStamps1.get_pixel(2, 2), delta=1e-5)
 
     def test_median_stamps_no_data(self):
         # Create a Trajectory that goes through the masked pixels.
-        trj = Trajectory()
-        trj.x = self.masked_x
-        trj.y = self.masked_y
-        trj.vx = 0
-        trj.vy = 0
+        trj = make_trajectory(x=self.masked_x, y=self.masked_y, vx=0.0, vy=0.0)
```

Comment:
Just to confirm, no concerns about int vs floats here? Correct?

Reply:
Correct. The data for velocities is stored as floats, so they were being converted before.